CMakeLists.txt

```diff
@@ -89,17 +89,11 @@ from the internet, we turn this option OFF." ON)
 set(SIMBODY_HOME $ENV{SIMBODY_HOME} CACHE
     PATH "The location of the Simbody installation to use; you can change this. Set as empty to let CMake search for Simbody automatically.")
 
-# TODO rename to OPENSIM_COPY_DEPENDENCIES and handle BTK.
-option(OPENSIM_COPY_SIMBODY "Copy Simbody headers and libraries into the
-OpenSim installation. This way, you can treat OpenSim and Simbody as one
-package, and you only need to set environment variables (e.g., PATH,
-LD_LIBRARY_PATH, DYLD_LIBRARY_PATH) for OpenSim.  On Windows, this also copies
-Simbody dll's and exe's into the OpenSim build directory, so that you don't
-need to set PATH to run OpenSim tests and examples; this is done via the
-SimbodyFiles project. If OFF, you likely must set those environment variables
-for both OpenSim and Simbody. To make a distributable installation on macOS, 
-this should be set to ON to avoid using absolute paths for RPATHs to Simbody
-libraries." ON)
+option(OPENSIM_COPY_DEPENDENCIES "Copy Simbody and BTK into the
+OpenSim installation. This should be set to ON when making a relocatable
+distribution, and should be set to OFF when packaging for Homebrew, Debian,
+etc. On Linux and macOS: we use relative RPATHs when ON, and absolute RPATHs
+when OFF." ON)
 
 option(OPENSIM_PYTHON_STANDALONE "Make the Python package standalone, meaning
 the OpenSim (and Simbody and BTK) shared libraries it depends on are copied
```

Comment:
I think setting PYTHONPATH is very atypical; the assumption is that you get your python packages from a python package manager (pip, conda, etc.), not from some third party website (e.g., simtk.org). So even telling users to cd <opensim-install>/lib/python2.7/site-packages/ and sudo python setup.py install is atypical. The best thing would be to distribute the python package separately via pypi, conda (homebrew, debian, etc.). Even then, whether we copy the libosim*.(so|dylib) files is not uniform: for pypi, we would include the shared libraries, on conda, we would probably have a "opensim-core" or "opensim" package that contains the shared libraries, and then a separate "opensim-python". I introduced this variable to help handle these different scenarios. However, perhaps we should hide it for most users.

My suggestions:

Document the PYTHONPATH option in our README.md.
Mark OPENSIM_PYTHON_STANDALONE as advanced.